### PR TITLE
Update free-programming-books-ar.md

### PR DESCRIPTION
**Pull request by Adamkaram**
## What does this PR do?
Add some books for programing and sql DB from arabic authors

### Description
A book to help people learn about programming in arabic greate resource actually.

### Why is this valuable (or not)?

People can learn how to program.

### How do we know it's really free?

arab people posting thier books for free in general  

### For book lists, is it a book? For course lists, is it a course? etc.
Books 

## Checklist:
- [✓ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ✓] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [✓ ] Include author(s) and platform where appropriate.
- [ ✓] Put lists in alphabetical order, correct spacing.
- [ ✓] Add needed indications (PDF, access notes, under construction).
- [ ✓] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
Fix the linter errors.

**Review by Adamkaram — COMMENTED** [review]
test if linter working

**Review by Adamkaram — COMMENTED** [review]
swap all orders for lint error order

**Comment by Adamkaram** [comment]
> Fix the linter errors.
done

File: books/free-programming-books-ar.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 * [Arduino](#arduino)
 * [Artificial Intelligence](#artificial-intelligence)
+* [C++](#C++)
```
Comment (eshellman) [inline books/free-programming-books-ar.md:7]: look at other list to see how to link to c++

File: books/free-programming-books-ar.md [hunk 2]
```diff
@@ -4,9 +4,11 @@
...
 * [DB & DBMS](#db--dbms)
 * [HTML and CSS](#html-and-css)
 * [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
+* [Java](#Java)
```
Comment (eshellman) [inline books/free-programming-books-ar.md:11]:
```suggestion
* [Java](#java)
```

File: books/free-programming-books-ar.md [hunk 3]
```diff
@@ -32,6 +34,11 @@
 * [مدخل إلى الذكاء الاصطناعي وتعلم الآلة](https://academy.hsoub.com/files/17-%D9%85%D8%AF%D8%AE%D9%84-%D8%A5%D9%84%D9%89-%D8%A7%D9%84%D8%B0%D9%83%D8%A7%D8%A1-%D8%A7%D9%84%D8%A7%D8%B5%D8%B7%D9%86%D8%A7%D8%B9%D9%8A-%D9%88%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%A2%D9%84%D8%A9/) - Mohamed Lahlah&rlm; (PDF)
 
 
+### C++
+
+* [لغة ++C ](https://ketabpedia.com/%D8%AA%D8%AD%D9%85%D9%8A%D9%84/%D8%B3%D9%84%D8%B3%D9%84%D8%A9-%D8%A7%D9%84%D8%A8%D8%B1%D9%85%D8%AC%D8%A9-%D8%A7%D9%84%D9%85%D9%81%D8%AA%D9%88%D8%AD%D8%A9-%D8%A8-c/) - (PDF) elBraa lRamly
```
Comment (eshellman) [inline books/free-programming-books-ar.md:39]: link directly to the book

File: books/free-programming-books-ar.md [hunk 4]
```diff
@@ -48,6 +55,11 @@
 * [مختصر دليل لغات البرمجة](https://alyassen.github.io/Brief-guide-to-programming-languages-v1.2.4.pdf) - Ali Al-Yassen&rlm; (PDF)
 
 
+### Java
+
+* [سلسلة تعلم الجافا ](https://www.noor-book.com/%D9%83%D8%AA%D8%A7%D8%A8-%D8%B3%D9%84%D8%B3%D9%84%D8%A9-%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%AC%D8%A7%D9%81%D8%A7-1-pdf) - (PDF) Mahmoud safy
```
Comment (eshellman) [inline books/free-programming-books-ar.md:60]: from internet archive. use internet archive link

File: books/free-programming-books-ar.md [hunk 5]
```diff
@@ -100,6 +112,7 @@
 
 ### SQL
 
+* [قواعد البيانات](https://www.noor-book.com/%D9%83%D8%AA%D8%A7%D8%A8-%D9%82%D9%88%D8%A7%D8%B9%D8%AF-%D8%A7%D9%84%D8%A8%D9%8A%D8%A7%D9%86%D8%A7%D8%AA-pdf) - (pdf) jomaa arifa
```
Comment (eshellman) [inline books/free-programming-books-ar.md:115]: from internet archive. use internet archive link

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
please use links from the original source; none of thes links lead to a book

**Comment by Adamkaram** [comment]
> please use links from the original source; none of thes links lead to a book

noorbook was the original source < authors them self have been posting it there

**Pull request closed without merging**